src/common/gui/SurgeBitmaps.cpp

```diff
@@ -86,6 +86,11 @@ void SurgeBitmaps::addEntry(int id)
 
 #ifdef USE_SCALABLE_BITMAPS
    VSTGUI::CBitmap *bitmap = new CScalableBitmap(CResourceDescription(id));
+#elif __linux__
+   char filename [1024];
+   snprintf (filename, 1024, "./resources/bitmaps/bmp%05d.png", id);
+   filename[1023] = '\0';
+   VSTGUI::CBitmap* bitmap = new VSTGUI::CBitmap(IPlatformBitmap::createFromPath( filename ));
```

Comment:
std::string() and c_str().

Reply:
how do you string format it? we need the `%05d` bit..

Reply:
Did some experimenting and a tes program:

```
#include <iomanip>
#include <iostream>
#include <sstream>

int main(int argc, char **argv)
{
    if (argc < 2)
    {
        std::cout << "Usage: " << argv[0] << "<number 1> ... <number N>"
                  << std::endl;
    }

    for (int i = 1; i < argc; i++)
    {
        int num = std::stoi(argv[i]);
        if (num >= 0 && num < 99999) {
            std::stringstream ss;
            ss << std::setw(5) << std::setfill('0') << num;
            std::cout << ss.str() << std::endl;
        }
    }

    return 0;
}
```

Sample output:

```
$ ./test 123 456 78901 23
00123
00456
78901
00023
```

Reply:
neat, though a bit overkill.
this is temporary code that is going to be replaced soon, once we load resources from memory...